src/sgame/sg_client.cpp

```diff
@@ -750,6 +750,18 @@ static void G_ClientCleanName( const char *in, char *out, size_t outSize, gclien
 		invalid = true;
 	}
 
+	if ( !(client->ent()->r.svFlags & SVF_BOT) )
+	{
+		const char *find = "[bot]";
+		const char *found;
+		const char *out_str;
```

Comment:
Doesn't follow variable naming convention

Reply:
I mean it follows the existing convention of  the  function (`out_string`)